cinema/views.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework import status
+
+from cinema.models import Movie
+from cinema.serializer import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movies_list_create(request: Request) -> Response:
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    serializer = MovieSerializer(data=request.data)
+    if serializer.is_valid(raise_exception=True):
+        if serializer.save():
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
1. No need for "if serializer.is_valid". It'll raise error if you just pass raise_exception=True
2. No need for serializer.save(). If serializer is valid, then data is correct, thus data'll be saved